website/contact/index.html

```diff
@@ -0,0 +1,93 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "_includes/base-resp.html" %}
+
+{% block page_title %}{{ _('Contact Us') }}{% endblock %}
+
+{% block page_desc %}
+{{ _('How to contact Thunderbird.') }}
+{% endblock %}
+
+{% block body_id %}contact{% endblock %}
+
+{% block page_css %}
+<link href="/media/css/thunderbird-features.css" rel="stylesheet" type="text/css"/>
+{% endblock %}
+
+{% block content %}
+<header id="main-feature">
+    <h1>{{ self.page_title() }}</h1>
+    <h2>{{ self.page_desc() }}</h2>
+</header>
+
+<nav role="directory">
+    <ul class="feature-links">
+        <li><a href="#support">{{ _('Get Support') }}</a></li>
+        <li><a href="#press"</a>{{ _('Press Inquiries')}}
+        <li><a href="#partnerships">{{ _('Partner with Thunderbird') }}</a></li>
+    </ul>
+</nav>
+
+<main role="main">
+    <section id="support">
+        <h2>{{ _('Get Support') }}</h2>
+        <div class="row">
+            <div class="col full">
+                <p>{{ _('How to get support for Thunderbird.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('Mozilla Support Website') }}</h3>
+                    <p>{{ _('The best place to start looking for answers if you have questions about Thunderbird is the Mozilla Support website, <a href="%(support)s">specifically the Thunderbird page</a>.
+                      There you can search for articles, ask questions, and read tutorials.')|format(support='https://support.mozilla.org/products/thunderbird') }}</p>
```

Comment:
This should use the url shortcut that already exists for the support site. `format(support=url('support'))`.